api/src/plugins/security.ts

```diff
@@ -0,0 +1,22 @@
+import { FastifyPluginCallback } from 'fastify';
+import fp from 'fastify-plugin';
+
+const fastifySentry: FastifyPluginCallback = (fastify, _options, done) => {
+  // OWASP recommended headers
+  fastify.addHook('onSend', async (_request, reply, payload) => {
+    void reply
+      .header('Cache-Control', 'no-store')
+      .header('Content-Security-Policy', "frame-ancestors 'none'")
+      .header('Content-Type', 'application/json; charset=utf-8')
+      // TODO: Increase this gradually to 2 years. Also, check if this needs to
+      // be off for dev
+      .header('Strict-Transport-Security', 'max-age=300; includeSubDomains')
```

Comment:
A quick poke around the internet suggests that yes, turning this off in development will save contributors some headache.

There's workarounds potentially, but seems like unnecessary overhead to fall back on those when we could set this conditionally.

Reply:
Thanks, Naomi. I'll stop being lazy and fix it!